wire-tests-swift/manifest/module_two/SwiftModuleTwoMessage.swift

```diff
@@ -8,6 +8,7 @@ public struct SwiftModuleTwoMessage {
 
     @ProtoDefaulted
     public var name: String?
+    public var order: module_one.SortOrder?
```

Comment:
Without this fix this would output `public var order: SortOrder?` which fails to compile.